manifests/0000_70_dns-operator_02-deployment.yaml

```diff
@@ -54,6 +54,10 @@ spec:
           capabilities:
             drop:
             - ALL
+          readOnlyRootFilesystem: true
+        volumeMounts:
+        - mountPath: /tmp
+          name: tmp-dir
```

Comment:
Does the dns-operator container really need a writable `/tmp`, or is this change the result of applying a generic change to add a writable `/tmp` in every container?

Reply:
It's the latter, we've been adding the tmp mounts even when tests pass since somethings silently need to write to /tmp for various reasons.